integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -935,20 +978,20 @@ public void testDynamicServerLifeCycleWithoutAdmin() {
 
   /**
    * Refer JIRA https://jira.oraclecorp.com/jira/browse/OWLS-86251
-   * Refer JIRA https://jira.oraclecorp.com/jira/browse/OWLS-86407
    * Once the admin server is stopped, operator can not start a new managed 
-   * server from scrtach if it has never been started eralier with Admin Server
-   * Once the admin server is stopped, the managed server can only be started 
-   * in MSI (managed server independence) mode. To start a manged server in 
-   * MSI mode, the pre-requisite is the that the manged server MUST be started 
-   * once before admin server is shutdown, so that the embedded LDAP server is 
-   * replicated from admin server to the managed server. 
-   * In this case of MII and DomainInImage model, the server state/configuration
-   * is not saved once the server is shutdown unless we use domain-on-pv model
-   * So in MII case, startServer.sh script update the replica count but the 
-   * server startup is defered till we re-start the adminserver. Here the 
-   * operator tries to start the managed server but it will keep on failing 
-   * until AdminServer is available.   
+   * server from scratch if it has never been started eralier with
```

Comment:
```suggestion
   * server from scratch if it has never been started earlier with
```

Reply:
what is the difference between testDynamicServerLifeCycleWithoutAdmin and testConfiguredServerLifeCycleWithoutAdmin? is it just the cluster type?

Reply:
yes. You are right

Reply:
Is there any reason for testing the same use case for both dynamic cluster and configured cluster? I don't think there is any difference in behavior based on cluster type.

Reply:
Yes, we need coverage on both configured and dynamic cluster as far as server lifecycle is concerned.